src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -265,6 +244,15 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     `a last known address exists`(2, AddressSource.MANUAL)
   }
 
+  @State(
+    "a recall with notes exists",
+  )
+  fun `a recall with notes exists`() {
+    `a user and a fully populated recall without documents exists`()
+    `a note exists`(1, matchedNoteId, matchedDocumentId)
+    `a note exists`(2)
+  }
+
```

Comment:
Do we have a need for this?  We cant delete notes so not sure of the use case

Reply:
Does the UI not add a note to a recall, and verify the response? Maybe not as it just does a single call (e.g. POST, not POST and then get recall)